### PR TITLE
add Splot reference

### PR DESCRIPTION
**Pull request by slumnitz**
I'd like to add splot to the geospatial viz tools. And include NSF + image as our sponsors. Please let me know if the logo and splot addition work. Thank you for organising this page, it's great!

### CODE REVIEW

**Comment by jbednar** [comment]
Thanks; I'll check it out!

File: tools/sponsors.yml [hunk 1]
```diff
@@ -57,3 +57,8 @@ unidata:
   label: Unidata
   url: https://www.unidata.ucar.edu
   logo: _static/badges/unidata.png
+
+nsf:
+  label: NSF
+  url: https://www.nsf.gov
+  logo: _static/badges/NSF.jpg
```
Comment (jbednar) [inline tools/sponsors.yml:64]:
```suggestion
  logo: _static/badges/NSF.jpg

```

File: tools/tools.yml [hunk 2]
```diff
@@ -212,6 +212,13 @@
       site: https://python-visualization.github.io/folium
       conda_channel: conda-forge
       builton: leaflet
+      
+    - repo: pysal/splot
+      sponsors: [nsf]
+      site: https://splot.readthedocs.io/en/latest/?badge=latest
```
Comment (jbednar) [inline tools/tools.yml:218]:
```suggestion
      site: https://splot.readthedocs.io
```

**Comment by slumnitz** [comment]
Hello @jbednar, thank you for your help and clarifying the workings of the sponsors logo! @sjsrey do you think the center would fulfil the mentioned requirements? Otherwise, I think we should just leave the sponsors logo out.

**Comment by sjsrey** [comment]
> Hello @jbednar, thank you for your help and clarifying the workings of the sponsors logo! @sjsrey do you think the center would fulfil the mentioned requirements? Otherwise, I think we should just leave the sponsors logo out.

Following @jbednar explanation, I think it would make sense to have UCR-CGS as the sponsor. In addition to the NSF grants being there, UCR is providing institutional support for the Center and its mission to maintain splot and all of PySAL.

**Comment by jbednar** [comment]
Sounds good!

**Comment by slumnitz** [comment]
@jbednar I added the cgs logo. I am not sure if two logo's can be displayed together? I might be able to just create a new .svg or .png with cgs and NSF side by side if that is easier. Let me know, happy to help!

**Comment by jbednar** [comment]
Thanks. It should just be cgs, not NSF; if someone were to contact NSF about splot they would have no idea what to answer. :-)

**Comment by slumnitz** [comment]
Ok sounds reasonable, sorry I took the meaning of "In addition" to literal for the logos.

**Comment by jbednar** [comment]
This change has been merged, but it won't show up on the public site until the build passes. The build is broken because some download counts are missing (see https://travis-ci.org/github/pyviz/pyviz.org), which I'm told will be addressed early next week, so we'll need to trigger a rebuild at that time to see this show up. (Or we could change the script to fetch the previous month's data if this month is missing, but that's more difficult than simply waiting a few days! :-)

**Comment by jbednar** [comment]
Ok, the builds are passing now, thanks to @sophiamyang, so splot is finally up on [pyviz.org/tools](pyviz.org/tools) .  Normally this would be much quicker, but thanks to all who helped in any case!

**Comment by slumnitz** [comment]
@jbednar thank you very much for your help and this awesome project! :)

**Pull request merged**